ament_copyright/ament_copyright/parser.py

```diff
@@ -175,7 +175,7 @@ def search_copyright_information(content):
     year = '\d{4}'
     year_range = '%s-%s' % (year, year)
     year_or_year_range = '(?:%s|%s)' % (year, year_range)
-    pattern = '^[^\n\r]?\s*Copyright(?:\s+\(c\))?\s+(%s(?:,\s*%s)*),?\s+([^\n\r]+)$' % \
+    pattern = '^[^\n\r]?[\s\*\w]*Copyright(?:\s+\(c\))?\s+(%s(?:,\s*%s)*),?\s+([^\n\r]+)$' % \
```

Comment:
This is because of things like

```
/*
 * My Amazing Project
 * Copyright (c) 2017
```

I saw a lot of that in some old willow code (like GMapping).